contracts/ERC4626/VenusERC4626.sol

```diff
@@ -72,108 +56,106 @@ contract VenusERC4626 is ERC4626, Ownable {
-        emit ClaimRewards(xvsBalance);
-    }
-
-    /// @notice Sets the paused state of a specific vault action.
-    /// @dev This function can only be called by the contract owner.
-    /// @param action The vault action to pause or unpause.
-    /// @param state The new paused state (true to pause, false to unpause).
-    /// Emits a {VaultActionPauseUpdated} event.
-    function setVaultActionPaused(VaultAction action, bool state) external onlyOwner {
-        _vaultPaused[action] = state;
-        emit VaultActionPausedUpdated(action, state);
     }
 
     /// @inheritdoc ERC4626
     function deposit(uint256 assets, address receiver) public override returns (uint256) {
-        _checkVaultActionPaused(VaultAction.DEPOSIT);
         if (assets > maxDeposit(receiver)) {
             revert ERC4626__DepositMoreThanMax();
         }
 
         uint256 shares = previewDeposit(assets);
         _deposit(_msgSender(), receiver, assets, shares);
-        afterDeposit(assets, shares);
+        afterDeposit(assets);
         return shares;
     }
 
     /// @inheritdoc ERC4626
     function mint(uint256 shares, address receiver) public override returns (uint256) {
-        _checkVaultActionPaused(VaultAction.MINT);
         if (shares > maxMint(receiver)) {
             revert ERC4626__MintMoreThanMax();
         }
 
         uint256 assets = previewMint(shares);
         _deposit(_msgSender(), receiver, assets, shares);
-        afterDeposit(assets, shares);
+        afterDeposit(assets);
         return assets;
     }
 
     /// @inheritdoc ERC4626
-    function withdraw(uint256 assets, address receiver, address withdrawer) public override returns (uint256) {
-        _checkVaultActionPaused(VaultAction.WITHDRAW);
-        if (assets > maxWithdraw(withdrawer)) {
+    function withdraw(uint256 assets, address receiver, address owner) public override returns (uint256) {
+        if (assets > maxWithdraw(owner)) {
             revert ERC4626__WithdrawMoreThanMax();
         }
 
         uint256 shares = previewWithdraw(assets);
-        beforeWithdraw(assets, shares);
-        _withdraw(_msgSender(), receiver, withdrawer, assets, shares);
+        beforeWithdraw(assets);
+        _withdraw(_msgSender(), receiver, owner, assets, shares);
         return shares;
     }
 
     /// @inheritdoc ERC4626
     function redeem(uint256 shares, address receiver, address redeemer) public override returns (uint256) {
-        _checkVaultActionPaused(VaultAction.REDEEM);
         if (shares > maxRedeem(redeemer)) {
             revert ERC4626__RedeemMoreThanMax();
         }
 
         uint256 assets = previewRedeem(shares);
-        beforeWithdraw(assets, shares);
+        beforeWithdraw(assets);
         _withdraw(_msgSender(), receiver, redeemer, assets, shares);
         return assets;
     }
 
+    function totalAssets() public view virtual override returns (uint256) {
+        return (VTOKEN.balanceOf(address(this)) * VTOKEN.exchangeRateStored()) / 1e18;
```

Comment:
What if we used `EXP_SCALE` from `lib/constants.sol` instead of hardcoding 1e18 in this file?